inst/WORDLIST

```diff
@@ -1,18 +1,29 @@
-
-altExp
 Alevin
+altExp
+altexp
+barcode
 CMD
+colData
+coldata
 commenter
 Dockerfile
+Ensembl
+ensembl
 Kallisto
 nf
 pre
-SCE
-SCEs
 PRs
+rowData
+rowdata
+rowname
```

Comment:
I think we can get rid of this with the edits you made? (I can't find any `rowname` in the docs, except for the one I removed)
```suggestion
```